frontend/public/locales/en/translation.json

```diff
@@ -112,6 +112,10 @@
   },
   "tabsBox": {
     "errorOnLoading": "An error occurred while loading the results.",
+    "rateLater": "Your rate-later videos will appear here. You can add some to your list by clicking on the '+' sign on the video cards. You can also add them directly from the extension.",
```

Comment:
Instead of displaying a button "Install the extension", we can make the description container shorter by using a simple HTML link directly in the paragraph. This will be less intrusive with users having already installed the extension.

Example of a HTML link used inside a translation: 

https://github.com/tournesol-app/tournesol/blob/e925beb7408eafb684717c2d6fb335f6c49695d6/frontend/src/pages/about/TermsOfService/TermsOfService.tsx#L47